versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 2.0"
+      version = ">= 4.0"
```

Comment:
Looks like this is also available in 3.x

https://registry.terraform.io/providers/hashicorp/aws/3.75.2/docs/data-sources/db_instance#db_name

Let's keep this as low as possible

Please also add this to `examples/complete/versions.tf`

Reply:
Sorry I said to use 4.x before. I was mistaken.